Description: Disable and fix tests without proper fixtures
Last-Update: 2022-09-02
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/stream/tests.rs
+++ b/src/stream/tests.rs
@@ -174,3 +174,4 @@
 
-#[test]
+//#[test]
+#[cfg(has_example_txt_zst)]
 fn test_cli_compatibility() {
@@ -190,3 +191,3 @@
 #[cfg(feature = "legacy")]
-#[test]
+//#[test]
 fn test_legacy() {
@@ -235,3 +236,4 @@
     // Where could I find some long text?...
-    let data = include_bytes!("../../zstd-safe/zstd-sys/src/bindings_zstd.rs");
+    // Debian: originally it's the prebuilt bindings, but we excluded it.
+    let data = include_bytes!("raw.rs");
     // Test a few compression levels.
